tools/nl/embeddings/sheets/embeddings_us_filtered.csv

```diff
@@ -1361,4 +1361,11 @@ Number,Name,Id,Description Strings,PaLM Generated Alternatives,ChatGPT descripti
 ,,Count_VolcanicAshEvent,,,volcanic eruptions; volcanic ash events; volcanic outbursts; volcanic activity; how many times has a volcano exploded,Volcano Events,,volcanic eruptions; volcanic ash events; volcanic outbursts; volcanic activity; how many times has a volcano exploded;Volcano Events;
 ,,dc/topic/Literacy,,,population's ability to read and write; literacy rate; how many people can read or not; how well read is the population; level of literacy; how many people cannot read or write; where can i find the most well read population in,Literacy,,population's ability to read and write; literacy rate; how many people can read or not; how well read is the population; level of literacy; how many people cannot read or write; where can i find the most well read population in;Literacy;
-,,dc/topic/RacialBreakdown,,,racial breakdown of a population; tell me about the demographics; ethnicity; demographic breakdown; population race; i want to know about the racial makeup,Racial Breakdown,,racial breakdown of a population; tell me about the demographics; ethnicity; demographic breakdown; population race; i want to know about the racial makeup;Racial Breakdown;
+,,dc/topic/RacialBreakdown,,,racial breakdown of a population; tell me about the demographics; ethnicity; demographic breakdown; population race; i want to know about the racial makeup; racial diversity of population; ethnic breakdown; how diverse are the demographics; demographic distribution,Racial Breakdown,,racial breakdown of a population; tell me about the demographics; ethnicity; demographic breakdown; population race; i want to know about the racial makeup; racial diversity of population; ethnic breakdown; how diverse are the demographics; demographic distribution;Racial Breakdown;
+,,dcid:sdg/g/SDG_1,End poverty; End poverty in all its forms; SDG end poverty; SDG poverty;,,,,,End poverty; End poverty in all its forms; SDG end poverty; SDG poverty;;
+,,dcid:sdg/g/SDG_2,Zero hunger; End hunger; Eradicate hunger; food security; sustainable agriculture; SDG hunger; SDG food; SDG sustainable agriculture;,,,,,Zero hunger; End hunger; Eradicate hunger; food security; sustainable agriculture; SDG hunger; SDG food; SDG sustainable agriculture;;
+,,dcid:sdg/g/SDG_3,Good health and well being; healthy lives; SDG good health; SDG healthy life; SDG health; SDG well being;,,,,,Good health and well being; healthy lives; SDG good health; SDG healthy life; SDG health; SDG well being;;
+,,dcid:sdg/g/SDG_4,Quality education; inclusive education; life long learning; SDG education; SDG learning; SDG quality and inclusive education,,,,,Quality education; inclusive education; life long learning; SDG education; SDG learning; SDG quality and inclusive education;
```

Comment:
All these should be dc/topic/SDG_4, etc.

Reply:
done!